src/components/map/hooks/use-trigger-camera-move-effect.tsx

```diff
@@ -158,10 +148,32 @@ export const moveCameraToCoordinate = (
     longitude: centerCoordinate.longitude + displacement,
     latitude: centerCoordinate.latitude + displacement,
   };
-  fitBounds(northEast, southWest, mapCameraRef, [
-    DEFAULT_PADDING,
-    DEFAULT_PADDING,
-    DEFAULT_PADDING + paddingBottom,
-    DEFAULT_PADDING,
-  ]);
+  fitBounds(northEast, southWest, mapCameraRef, padding);
+};
+
+const useCalculatePaddings = (): MapboxGL.Padding => {
+  const {height: bottomSheetHeight} = useBottomSheet();
+  const {minHeight: tabBarMinHeight} = useBottomNavigationStyles();
+  const {height: screenHeight} = Dimensions.get('screen');
+  const padding = screenHeight * 0.1;
+
+  if (Platform.OS === 'android') {
+    const headerHeight = StatusBar.currentHeight ?? 0;
+
+    // This is the base level for the padding, where the values are divided by the device font scale to match the map unit.
+    const scaledBottomSheetPadding = bottomSheetHeight - tabBarMinHeight;
+    return [
+      padding + headerHeight, // Subtract the header height on Android since it is not see through
+      padding,
+      padding + scaledBottomSheetPadding,
+      padding,
+    ].map((p) => p / PixelRatio.getFontScale()) as MapboxGL.Padding;
```

Comment:
@jorelosorio do you remember why you are dividing on the font scale here?

Reply:
As far I remember, the scales inside the map are not based on dpi so by using / by that factor becomes somehow pixel units, not point units.